lib/modules/manager/hermit/index.ts

```diff
@@ -0,0 +1,15 @@
+import { HermitDatasource } from '../../datasource/hermit';
+import { id as versionId } from '../../versioning/hermit';
+import { updateArtifacts } from './artifacts';
+import { defaultConfig as partialDefaultConfig } from './default-config';
+import { extractPackageFile } from './extract';
+import { updateDependency } from './update';
+
+export { extractPackageFile, updateDependency, updateArtifacts };
+
+export const defaultConfig = {
+  ...partialDefaultConfig,
```

Comment:
don't add non renovate options

Reply:
sorry, I'm not quite sure what you mean here, do you want the config to be specific? like

```
export const defaultConfig = {
  fileMatch: partialDefaultConfig.fileMatch,
  excludeCommitPaths: partialDefaultConfig.excludeCommitPaths,
  versioning: versionId,
}
```